cluster/proxy_service.py

```
@@ -0,0 +1,267 @@
+"""HTTP proxy service that forwards requests to NATS JetStream"""
+
+import asyncio
```

Ruff (0.12.2)

`asyncio` imported but unused

Remove unused import: `asyncio`

(F401)